secure_fw/core/tfm_core_mem_check.c

```diff
@@ -35,6 +35,28 @@ static enum tfm_status_e has_access_to_region(const void *p, size_t s,
     range_access_allowed_by_mpu =
                           cmse_check_address_range((void *)p, s, flags) != NULL;
 
+    if (!range_access_allowed_by_mpu) {
+    	/*
+    	 * Verification failure may be due to address range crossing
```

Comment:
Why does it fail if it crosses an IDAU boundary? Does it always fail when crossing an IDAU boundary?